Content.Server/Atmos/EntitySystems/AtmosphereSystem.cs

```diff
@@ -107,4 +121,9 @@ public override void Update(float frameTime)
 
         _exposedTimer -= ExposedUpdateDelay;
     }
+
+    private void CacheDecals()
+    {
+        _burntDecals = _prototypeManager.EnumeratePrototypes<DecalPrototype>().Where(x => x.Tags.Contains("burnt")).Select(x => x.ID).ToList();
+    }
 }
```

Comment:
ToArray because you're never actually modifying this and they're faster than lists.